### PR TITLE
feat(mls): add toggle to create mls groups FS-539

### PR DESCRIPTION
**Pull request by johnxnguyen**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/FS-539" title="FS-539" target="_blank">Sub-taskFS-539</a>  [iOS] Add UI option for MLS in create conversation
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

- Add toggle to create an MLS conversation (toggle doesn't do anything just yet), the toggle is only visible if the self user is allowed to (depends on a feature config)
- Add a developer flags screen that is used to force showing the toggle or not

### Dependencies (Optional)

Needs releases with:

- https://github.com/wireapp/wire-ios-data-model/pull/1368
- https://github.com/wireapp/wire-ios-request-strategy/pull/519

#### Test Coverage

- No tests added because this is a temporary toggle.

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: Wire-iOS/Sources/Developer/DeveloperFlagsView.swift [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/Developer/DeveloperFlagsView.swift:19]: 😄

**Comment by KaterinaWire** [comment]
@johnxnguyen Do we need new snapshot tests?

**Comment by johnxnguyen** [comment]
> Do we need new snapshot tests?

@KaterinaWire I don't think so, since we'll remove this toggle eventually. But I'm happy to write one if you think otherwise.

**Review by KaterinaWire — APPROVED** [review]

File: Cartfile.resolved [hunk 2]
```diff
@@ -18,14 +18,14 @@ github "wireapp/swift-protobuf" "1.15.0_xcframework"
...
+github "wireapp/wire-ios-data-model" "feat/mls"
 github "wireapp/wire-ios-images" "41.0.0"
 github "wireapp/wire-ios-link-preview" "35.0.0"
 github "wireapp/wire-ios-notification-engine" "7.0.0"
 github "wireapp/wire-ios-protos" "30.1.1"
-github "wireapp/wire-ios-request-strategy" "267.0.0"
+github "wireapp/wire-ios-request-strategy" "feat/mls"
 github "wireapp/wire-ios-share-engine" "248.0.0"
-github "wireapp/wire-ios-sync-engine" "425.0.0"
+github "wireapp/wire-ios-sync-engine" "feat/mls-feature-config"
```
Comment (johnxnguyen) [inline Cartfile.resolved:28]:
```suggestion
github "wireapp/wire-ios-sync-engine" "feat/mls"
```

**Review by agisilaos — APPROVED** [review]

File: Wire-iOS.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -1239,7 +1239,6 @@
 		EE01D08825ADA3D5001DB205 /* AppLockModule.Interactor.swift in Sources */ = {isa = PBXBuildFile; fileRef = EE01D08325ADA3D5001DB205 /* AppLockModule.Interactor.swift */; };
 		EE01D08925ADA3D5001DB205 /* AppLockModule.View.swift in Sources */ = {isa = PBXBuildFile; fileRef = EE01D08425ADA3D5001DB205 /* AppLockModule.View.swift */; };
 		EE09DE6025B24B2000B515F0 /* AppLockModuleRouterTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = EE09DE5E25B24B2000B515F0 /* AppLockModuleRouterTests.swift */; };
-		EE12922827E47E6D00C9CC60 /* Wire Share Extension.appex in Embed App Extensions */ = {isa = PBXBuildFile; fileRef = 168A16A91D9597C2005CFA6C /* Wire Share Extension.appex */; settings = {ATTRIBUTES = (RemoveHeadersOnCopy, ); }; };
```
Comment (agisilaos) [inline Wire-iOS.xcodeproj/project.pbxproj:1242]: @johnxnguyen I don't know if we should remove this. Why this is happening when we add a new file to the directory?

File: Wire-iOS.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -7288,7 +7299,6 @@
 				EE373FA923FBF438008C1E52 /* ConversationInputBarViewControllerDelegateTests.swift */,
 				EFC57A0F21BABCC90008DBE6 /* ConversationInputBarViewControllerAudioRecorderSnapshotTests.swift */,
 				061275D626F23535006E8D4C /* ConversationInputBarViewController+DropInteractionTests.swift */,
-				70C5560C27B2E6B100119A7D /* MockClassificationProvider.swift */,
```
Comment (agisilaos) [inline Wire-iOS.xcodeproj/project.pbxproj:7291]: Same here.

File: Wire-iOS.xcodeproj/project.pbxproj (hunk 3 above)
Comment (johnxnguyen) [reply Wire-iOS.xcodeproj/project.pbxproj:1242]: I don't know, but I see this in almost every PR.

**Pull request merged**